test/google_crawler_web/features/keywords/filter_keyword_test.exs

```diff
@@ -46,4 +48,23 @@ defmodule GoogleCrawlerWeb.FilterKeywordTest do
     |> assert_has(css(@selectors[:keyword_table_cell], text: keyword2.title))
     |> refute_has(css(@selectors[:keyword_table_cell], text: keyword3.title))
   end
+
+  feature "filters keywords which search results match amount and operation by the given params", %{session: session} do
+    user = insert(:user)
+    keyword1 = insert(:keyword, user: user)
+    keyword2 = insert(:keyword, user: user)
+
+    insert(:search_result, keyword: keyword1)
+    insert(:search_result, keyword: keyword1)
```

Comment:
Double `search_result` for keyword1, was that intentional?

Reply:
yes, to create a keyword with 2 search result and this test is filter keyword which contains 2 search result